xbmcaddon.py

```diff
@@ -158,16 +164,37 @@ def getAddonInfo(self, id):  # NOSONAR
 
         Possible options are: author, changelog, description, disclaimer, fanart, icon, id,
                               name, path, profile, stars, summary, type, version
-        """ 
-
-        if id == "path":
-            return self.__add_on_path
+        """
+        id = id.lower()
+
+        if id == "author":
+            return self.__author
+        elif id == "changelog":
+            return self.__news
+        elif id == "description":
+            return self.__description
+        elif id == "disclaimer":
+            return self.__disclaimer
+        elif id == "fanart":
+            return self.__fanart
+        elif id == "icon":
+            return self.__icon
         elif id == "id":
             return self.__add_on_id
-        elif id == "version":
-            return self.__version
         elif id == "name":
             return self.__name
+        elif id == "path":
+            return self.__add_on_path
+        elif id == "profile":
+            return self.__add_on_profile_path
+        elif id == "stars":
```

Comment:
With the `pass` this one makes little sense. So I guess we should either return a fixed value, or remove this `if`-clause.

Reply:
I made that to clearly identify that was missing, but maybe i wold have raised an "unsupported" exception.
I will remove.